.github/workflows/team-workflow.yaml

```
@@ -0,0 +1,52 @@
+name: Deploy to Google Cloud Run
+
+on:
+  push:
+    branches:
+      - main  # Run the workflow on push to the main branch
```

```suggestion
  push:
    branches:
      - main  # Run the workflow on push to the main branch
  workflow_dispatch:
```

wenn workflow_dispatch drin hesch, chasch de workflow au manuell triggere im actions tab.

---

Done, thx!